core/src/sigverifier/bls_sigverifier.rs

```diff
@@ -88,6 +91,9 @@ impl SigVerifier for BLSSigVerifier {
                         stats_updater.received_malformed += 1;
                         continue;
                     };
+                    self.ag_metrics
+                        .write()
+                        .record_vote(*pubkey, vote, Duration::MAX);
```

Comment:
do we also want to record skips?

Reply:
I'm not sure if updating stats in sigverify makes the most sense. I'm wondering if we should add vote stats near `fn process_consensus_message`? I believe this would also allow us to observe our own votes, so we wouldn't need to call `record_vote` in two places